src/csharp/Grpc.Core/AsyncDuplexStreamingCall.cs

```diff
@@ -117,6 +117,10 @@ public Metadata GetTrailers()
         /// Otherwise, requests cancellation of the call which should terminate all pending async operations associated with the call.
         /// As a result, all resources being used by the call should be released eventually.
         /// </summary>
+        /// <remarks>
+        /// Normally, there is no need for you to dispose the call unless you want to utilize the
+        /// "Cancel" semantics of invoking <c>Dispose</c>.
+        /// </remarks>
         public void Dispose()
```

Comment:
If I'm understanding this correctly, dispose just wraps ```call.cancel()```.
I'm not sure that the added remarks add any clarity to what this method does, it seems pretty clear from the summary.

I think the confusion stems not from a lack of documentation, but rather c# idioms.
It seems like the .NET community leans towards calling Dispose on any object that implements IDisosable, but there isn't quite a consensus.

My personal opinion is that you should just expose a ```Cancel()``` method.  I've seen suggestions where ```Dispose()``` should be wrapped with domain-specific method names: https://msdn.microsoft.com/en-us/library/b1yfkh5e(v=vs.100).aspx